File: EarlGrey.Common/Weather/WindForecastData.cs
namespace EarlGrey.Common.Weather {
    public class WindForecastData {


        /// <summary>Wind speed minimum in KM/H</summary>
        public double? MinSpeed { get; set; }

        /// <summary>Wind speed maximum in KM/H</summary>
        public double? MaxSpeed { get; set; }

        /// <summary>Direction to where the wind is GOING in degrees</summary>
        public double? Direction { get; set; }

        /// <summary>Direction string (IE: NW, SW, N, Calm)</summary>
        public string DirectionString => WindDirection.DegreesToCompass(Direction);
    }
}

File: earlgrey.web/src/model/weather/WindForecastData.ts
export default interface WindForecastData {
    /** min speed in KM/H */
    minSpeed: number

    /**max speed in KM/H */
    maxSpeed: number

    /**Direction the wind is blowing towards */
    direction: number

    /** Direction cardinal direction (N, NNW, NW, WNW, W, etc.) */
    directionString: string
}